### DISCUSSION TITLE
We found the patch versions not appeared in Golang Index.

### DISCUSSION THREAD

**Discussion by Silence-worker-02** [grafana/grafana#71909] [General]
Hello, we are a research team working on Golang. During our investigation, we discovered that the following CVEs -CVE-2023-1410, CVE-2021-41244, CVE-2022-39306, CVE-2022-39307, CVE-2022-39201, CVE-2022-31130, CVE-2022-31107, CVE-2021-39226, CVE-2022-31123, CVE-2023-1387, CVE-2022-39229, CVE-2022-39328, CVE-2022-31097 were addressed and the patch versions were released. However, we noticed that these patch versions have not appeared in the Golang Index, which means that 'go list' cannot automatically push the patch versions to downstream users.

We recommend that after releasing the versions, you push them to the Golang Index using the command 'go get github.com/grafana/grafana@version'. This will enable the automatic distribution of the patch versions to downstream users. Thank you for your attention.

**Comment by gelicia** [comment]
Hello, as you may have heard, we are [transitioning away from using discussions to discuss feature requests](https://github.com/grafana/grafana/discussions/73424). We are migrating this discussion to an issue and closing the discussion. The issue is #82624. Feel free to continue the discussion around this there. Thank you!
